bin/git/update-hooks

```diff
@@ -19,11 +56,15 @@ install_hook_wrapper() {
         # If a hook already exists and is a file:
         if [ -f $HOOK_DIR/$hook ]; then
             step_name "Moving existing '$hook' script to '$hook-local'"
-            mv $HOOK_DIR/$hook $HOOK_DIR/$hook-local
+            if [ $make_updates == 1 ]; then
```

Comment:
I think it's just a string, so safer to do "$x == "1". Kind of why I prefer true/false over 1/0 - more self-documenting.